ArWikiCats/translations/nats/Nationality.py

```
@@ -112,27 +163,43 @@ def load_sources(
     raw_nats_as_en_key: Dict[str, Dict[str, str]] | None = None,
 ) -> Dict[str, NationalityEntry]:
     """
-    Load nationality JSON data and merge All_Nat_o + uu_nats + Sub_Nat.
+    Load nationality JSON data and merge nationalities_data + sub_nats.
     Ensures all entries follow the NationalityEntry structure (string values only).
+
     """
+    data_to_review = {
```

Ruff (0.14.10)

Local variable `data_to_review` is assigned to but never used

Remove assignment to unused variable `data_to_review`

(F841)